addons/udes_permissions/__manifest__.py

```diff
@@ -11,7 +11,10 @@
     "category": "Uncategorized",
     "version": "0.1",
     # any module necessary for this one to work correctly
-    "depends": ["base"],
+    "depends": [
+        "base",
+        "udes_hierarchical_tree_view",
```

Comment:
Might be worth considering making `udes_common` depend on `udes_hierarchical_tree_view`, and adding a dependency to `udes_common` instead.